### ISSUE TITLE
[1.15] Crash on Cube unload after rename to BigCube

### ISSUE THREAD

**Issue by AidanLovelace** [Barteks2x/CubicChunks#539]
#540 
```
java.util.concurrent.CompletionException: java.lang.BootstrapMethodError: java.lang.NoClassDefFoundError: io/github/opencubicchunks/cubicchunks/chunk/cube/Cube
	at java.util.concurrent.CompletableFuture.encodeThrowable(CompletableFuture.java:273) ~[?:1.8.0_251] {}
	at java.util.concurrent.CompletableFuture.completeThrowable(CompletableFuture.java:280) ~[?:1.8.0_251] {}
	at java.util.concurrent.CompletableFuture.uniApply(CompletableFuture.java:618) ~[?:1.8.0_251] {}
	at java.util.concurrent.CompletableFuture.uniApplyStage(CompletableFuture.java:628) ~[?:1.8.0_251] {}
	at java.util.concurrent.CompletableFuture.thenApply(CompletableFuture.java:1996) ~[?:1.8.0_251] {}
	at net.minecraft.world.server.ChunkHolder.processCubeUpdates(ChunkHolder.java:10340) ~[?:?] {re:mixin,pl:accesstransformer:B,re:classloading,pl:accesstransformer:B,pl:mixin:APP:cubicchunks.mixins.core.json:common.chunk.MixinChunkHolder,pl:mixin:APP:cubicchunks.mixins.access.json:common.ChunkHolderAccess,pl:mixin:APP:cubicchunks.mixins.asm.json:common.MixinAsmTarget,pl:mixin:A}
	at net.minecraft.world.server.ChunkHolder.handler$zzc000$processUpdates(ChunkHolder.java:619) ~[?:?] {re:mixin,pl:accesstransformer:B,re:classloading,pl:accesstransformer:B,pl:mixin:APP:cubicchunks.mixins.core.json:common.chunk.MixinChunkHolder,pl:mixin:APP:cubicchunks.mixins.access.json:common.ChunkHolderAccess,pl:mixin:APP:cubicchunks.mixins.asm.json:common.MixinAsmTarget,pl:mixin:A}
	at net.minecraft.world.server.ChunkHolder.processUpdates(ChunkHolder.java) ~[?:?] {re:mixin,pl:accesstransformer:B,re:classloading,pl:accesstransformer:B,pl:mixin:APP:cubicchunks.mixins.core.json:common.chunk.MixinChunkHolder,pl:mixin:APP:cubicchunks.mixins.access.json:common.ChunkHolderAccess,pl:mixin:APP:cubicchunks.mixins.asm.json:common.MixinAsmTarget,pl:mixin:A}
	at net.minecraft.world.server.ChunkHolder.processUpdatesCC(ChunkHolder.java) ~[?:?] {re:mixin,pl:accesstransformer:B,re:classloading,pl:accesstransformer:B,pl:mixin:APP:cubicchunks.mixins.core.json:common.chunk.MixinChunkHolder,pl:mixin:APP:cubicchunks.mixins.access.json:common.ChunkHolderAccess,pl:mixin:APP:cubicchunks.mixins.asm.json:common.MixinAsmTarget,pl:mixin:A}
	at net.minecraft.world.server.TicketManager.md966411$lambda$processUpdates$1$6(TicketManager.java:632) ~[?:?] {re:mixin,pl:accesstransformer:B,re:classloading,pl:accesstransformer:B,pl:mixin:APP:cubicchunks.mixins.core.json:common.ticket.MixinTicketManager,pl:mixin:APP:cubicchunks.mixins.access.json:common.TicketManagerAccess,pl:mixin:A}
	at java.lang.Iterable.forEach(Iterable.java:75) ~[?:1.8.0_251] {}
	at net.minecraft.world.server.TicketManager.handler$zzl000$processUpdates(TicketManager.java:632) ~[?:?] {re:mixin,pl:accesstransformer:B,re:classloading,pl:accesstransformer:B,pl:mixin:APP:cubicchunks.mixins.core.json:common.ticket.MixinTicketManager,pl:mixin:APP:cubicchunks.mixins.access.json:common.TicketManagerAccess,pl:mixin:A}
	at net.minecraft.world.server.TicketManager.processUpdates(TicketManager.java:134) ~[?:?] {re:mixin,pl:accesstransformer:B,re:classloading,pl:accesstransformer:B,pl:mixin:APP:cubicchunks.mixins.core.json:common.ticket.MixinTicketManager,pl:mixin:APP:cubicchunks.mixins.access.json:common.TicketManagerAccess,pl:mixin:A}
	at net.minecraft.world.server.ServerChunkProvider.func_217235_l(ServerChunkProvider.java:273) ~[?:?] {re:mixin,pl:accesstransformer:B,re:classloading,pl:accesstransformer:B,pl:mixin:APP:cubicchunks.mixins.core.json:common.MixinServerChunkProvider,pl:mixin:A}
	at net.minecraft.world.server.ServerChunkProvider.access$200(ServerChunkProvider.java:49) ~[?:?] {re:mixin,pl:accesstransformer:B,re:classloading,pl:accesstransformer:B,pl:mixin:APP:cubicchunks.mixins.core.json:common.MixinServerChunkProvider,pl:mixin:A}
	at net.minecraft.world.server.ServerChunkProvider$ChunkExecutor.driveOne(ServerChunkProvider.java:514) ~[?:?] {re:mixin,pl:accesstransformer:B,re:classloading,pl:accesstransformer:B}
	at net.minecraft.world.server.ServerChunkProvider.driveOneTask(ServerChunkProvider.java:269) ~[?:?] {re:mixin,pl:accesstransformer:B,re:classloading,pl:accesstransformer:B,pl:mixin:APP:cubicchunks.mixins.core.json:common.MixinServerChunkProvider,pl:mixin:A}
	at net.minecraft.server.MinecraftServer.driveOneInternal(MinecraftServer.java:747) ~[?:?] {re:mixin,pl:accesstransformer:B,re:classloading,pl:accesstransformer:B,pl:mixin:APP:cubicchunks.mixins.core.json:common.MixinMinecraftServer,pl:mixin:A}
	at net.minecraft.server.MinecraftServer.driveOne(MinecraftServer.java:736) ~[?:?] {re:mixin,pl:accesstransformer:B,re:classloading,pl:accesstransformer:B,pl:mixin:APP:cubicchunks.mixins.core.json:common.MixinMinecraftServer,pl:mixin:A}
	at net.minecraft.util.concurrent.ThreadTaskExecutor.driveUntil(ThreadTaskExecutor.java:123) ~[?:?] {re:classloading,pl:accesstransformer:B,re:mixin,pl:accesstransformer:B}
	at net.minecraft.server.MinecraftServer.runScheduledTasks(MinecraftServer.java:722) ~[?:?] {re:mixin,pl:accesstransformer:B,re:classloading,pl:accesstransformer:B,pl:mixin:APP:cubicchunks.mixins.core.json:common.MixinMinecraftServer,pl:mixin:A}
	at net.minecraft.server.MinecraftServer.run(MinecraftServer.java:666) ~[?:?] {re:mixin,pl:accesstransformer:B,re:classloading,pl:accesstransformer:B,pl:mixin:APP:cubicchunks.mixins.core.json:common.MixinMinecraftServer,pl:mixin:A}
	at java.lang.Thread.run(Thread.java:748) ~[?:1.8.0_251] {}
```

**Comment by NotStirred** [comment]
Ah, that's my bad, I'll fix it now

**Comment by AidanLovelace** [comment]
Oh I've got it fixed already

**Comment by AidanLovelace** [comment]
Making a PR now

**Comment by NotStirred** [comment]
why make an issue then x)

**Comment by AidanLovelace** [comment]
Seems like a crash is an issue even if I've just fixed it.

**Comment by NotStirred** [comment]
closed by afd033962e1c91a6636006f606dbd58911a48d94 and d748318fe23960ed183cb03fa0769a0a43a7054e
